task1/lib/main.dart

```diff
@@ -0,0 +1,157 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Task 1',
+      home: MainPage(),
+    );
+  }
+}
+
+class MainPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(centerTitle: true, title: Text('Flutter Vadodara')),
+      body: Container(
+        padding: EdgeInsets.only(top: 16, bottom: 16),
+        margin: EdgeInsets.fromLTRB(32, 0, 32, 0),
+        child: Column(
+          children: [
+            Container(
+                color: Colors.blue,
+                width: 250.0,
+                height: 100,
+                margin: EdgeInsets.all(30),
```

Comment:
Remove the margin and width parameter from every Container in the page.

Reply:
Kind of declare once for every container?

Reply:
It's not required right now.